vault/logical_system.go

```diff
@@ -1498,6 +1501,21 @@ func (b *SystemBackend) handleMount(ctx context.Context, req *logical.Request, d
 		config.DelegatedAuthAccessors = apiConfig.DelegatedAuthAccessors
 	}
 
+	if apiConfig.IdentityTokenKey != "" {
+		identityStore := b.Core.IdentityStore()
+		identityStore.oidcLock.RLock()
+		defer identityStore.oidcLock.RUnlock()
+		k, err := identityStore.getNamedKey(ctx, identityStore.view, apiConfig.IdentityTokenKey)
```

Comment:
I'm not sure `identityStore.view` is right here when using namespaces. Doing some testing on ent.

Reply:
Update that it's incorrect. Fixing this before merging.

Reply:
Fixed in https://github.com/hashicorp/vault/pull/24962/commits/2a409cfcfdfa26374029494f5e8b9800243715a3.